src/model/id.rs

```diff
@@ -172,6 +172,14 @@ impl RoleId {
     }
 }
 
+impl UserId {
+    /// Search the cache for the channel with the Id.
+    #[cfg(all(feature = "cache", feature = "methods"))]
+    pub fn find(&self) -> Option<User> {
+        CACHE.read().unwrap().get_user(*self).map(|x| x.clone())
+    }
```

Comment:
I really ought to separate users from guild member objects, but that's an optimization for another day..